### ISSUE TITLE
Custom build still shows all tags

### ISSUE THREAD

**Issue by lpares12** [mattiasw/ExifReader#408]
### Description

I've followed the instructions for creating a custom build and I can still see all tags of the image when parsing a File.

**Additional details**

- ExifReader version: 4.23.5
- Web browser and version: Chrome

**How to reproduce**

Added this to package.json:

```
"exifreader": {
    "include": {
      "jpeg": true,
      "png": true,
      "exif": [
        "GPSLatitude",
        "GPSLatitudeRef",
        "GPSLongitude",
        "GPSLongitudeRef",
        "GPSAltitude",
        "GPSAltitudeRef"
      ]
    }
  }
```

Run `npm uninstall exifreader` and `npm install exifreader` (also tried `npm rebuild exifreader` as explained in the docs). Cleared .vite file (also ran `npm cache clean --force`) and cleared browser cache.

**What I expected would happen:**

I'd expect the following code to just print the GPS exif tags:
```
<script lang="ts" setup>
import ExifReader from 'exifreader';

async function readExif(file: File) {
  const tags = await ExifReader.load(file);
  console.log(tags);
}
</script>
```

**What really happened:**

It prints all the other tags too: bits per sample, image height, image width, lens information, white balance, resolution,...

So I'm guessing the package hasn't been reduced in size.

**Comment by mattiasw** [comment]
Hi! What's the size of the rebuilt file `node_modules/exifreader/dist/exif-reader.js`? If it is smaller than ~85 KiB, try importing from `exifreader/dist/exif-reader.js` instead (it should pick it up with a regular import too but maybe vite does things differently).

**Comment by mattiasw** [comment]
It sounds like #409 could be the same issue you're seeing. The bundle size is not changing after a rebuild. We're trying to create a minimal reproducible example there.

**Comment by lpares12** [comment]
`node_modules/exifreader/dist/exif-reader.js` is 86KB. What do you mean by `try importing from exifreader/dist/exif-reader.js instead`?

I'm not sure what's the proper syntax for that. My idea marks all my tries as:

> Cannot find module '../node_modules/exifreader/dist/exif-reader.js' or its corresponding type declarations

As for #409, I'm also using Windows, might be it?

**Comment by mattiasw** [comment]
Then it's Windows. And I can reproduce that in a virtual machine myself so no need for you to do anything more right now. I'll get back to you both.

**Comment by mattiasw** [comment]
It was a path separator issue, of course... Fixed and published as version 4.23.7.
